### PR TITLE
Fix dashboard tests

### PR DESCRIPTION
**Pull request by somebody1234**
### Pull Request Description
Fixes issues that were causing CI to fail.

### Important Notes
None

### Checklist

Please ensure that the following checklist has been satisfied before submitting the PR:

- [x] ~~The documentation has been updated, if necessary.~~
- [x] ~~Screenshots/screencasts have been attached, if there are any visual changes. For interactive or animated visual changes, a screencast is preferred.~~
- [x] All code follows the
      [Scala](https://github.com/enso-org/enso/blob/develop/docs/style-guide/scala.md),
      [Java](https://github.com/enso-org/enso/blob/develop/docs/style-guide/java.md),
      and
      [Rust](https://github.com/enso-org/enso/blob/develop/docs/style-guide/rust.md)
      style guides. In case you are using a language not listed above, follow the [Rust](https://github.com/enso-org/enso/blob/develop/docs/style-guide/rust.md) style guide.
- All code has been tested:
  - [x] ~~Unit tests have been written where possible.~~
  - [x] ~~If GUI codebase was changed, the GUI was tested when built using `./run ide build`.~~

### CODE REVIEW

File: app/ide-desktop/lib/dashboard/playwright.config.ts [hunk 1]
```diff
@@ -16,8 +16,9 @@ export default test.defineConfig({
   repeatEach: process.env.CI ? 3 : 1,
   expect: {
     toHaveScreenshot: { threshold: 0 },
-    timeout: 30_000,
+    timeout: 120_000,
   },
+  timeout: 120_000,
```
Comment (farmaazon) [inline app/ide-desktop/lib/dashboard/playwright.config.ts]: I don't think increasing timeout will fix our problem, actually.

When I look at `drag labels onto multiple rows`, I see many events are emitted one after another, without waiting for any possible effect (selecting rows maybe?) - are we sure the next event will take desired effect before the previous one will have it?

For example, in https://github.com/enso-org/enso/blob/1a8b82e23778f8e6a9e092b76fc964fb2000e1c6/app/gui2/e2e/componentBrowser.spec.ts#L68 we click at the background to deselect any selected node. After every click, we actually have a check if all the nodes are deselected - without it, sometimes we proceed before the deselecting took place and the test was flaky.

Comment (somebody1234) [reply app/ide-desktop/lib/dashboard/playwright.config.ts]: you're right - nice catch, thanks!
i ran a bunch of tests when i was afk and got this a number of times:
```
Running 4 tests using 1 worker

  ✓  1 labels.spec.ts:5:6 › drag labels onto single row (5.5s)
  ✘  2 labels.spec.ts:37:6 › drag labels onto multiple rows (2.0m)
  ✓  3 labels.spec.ts:73:6 › drag (recursive) (7.2s)
  ✓  4 labels.spec.ts:175:6 › drag (inverted, recursive) (4.0s)


  1) labels.spec.ts:37:6 › drag labels onto multiple rows ──────────────────────────────────────────

    Test timeout of 120000ms exceeded.

    Error: expect(locator).toBeVisible()

    Locator: getByTestId('drive-view').getByRole('table').locator('tbody').getByRole('row').first().getByTestId('asset-label').getByText('aaaa')
    Expected: visible
    Received: hidden
    Call log:
      - expect.toBeVisible with timeout 120000ms
      - waiting for getByTestId('drive-view').getByRole('table').locator('tbody').getByRole('row').first().getByTestId('asset-label').getByText('aaaa')


      66 |   const assetRows = actions.locateAssetRows(page)
      67 |   const labels = actions.locateLabelsPanelLabels(page)
    > 68 |   const label = 'bbbb'
         |                                                     ^
      69 |   api.addLabel('aaaa', backend.COLORS[0])
      70 |   // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
      71 |   api.addLabel(label, backend.COLORS[1]!)

        at /home/me/git/enso/app/ide-desktop/lib/dashboard/e2e/labels.spec.ts:68:83

  1 failed
    labels.spec.ts:37:6 › drag labels onto multiple rows ───────────────────────────────────────────
  3 passed (2.3m)
```

i suppose i'll have to figure out how to properly wait then

Comment (somebody1234) [reply app/ide-desktop/lib/dashboard/playwright.config.ts]: nvm. it's not an issue with not waiting, it's an actual issue with the `onDrop` handler

**Review by indiv0 — APPROVED** [review]
LGTM

**Pull request merged**